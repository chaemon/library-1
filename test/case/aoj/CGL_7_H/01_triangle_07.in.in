3 5
3 11
-13 2
12 -13
