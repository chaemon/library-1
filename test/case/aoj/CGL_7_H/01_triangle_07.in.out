78.5215996561
